app/smc/pytest/e2e_smoke.py

```diff
@@ -249,15 +277,16 @@ def upgrade_from_version_test(
     ) == read_telem(0, TAG_CM_FW_VERSION)
 
 
-def test_upgrade_from_18_10(arc_chip_dut, tmp_path: Path, board_name, unlaunched_dut):
```

Comment:
Would it make sense to keep the old test and add the new test too? Otherwise, `replace_bootloader` would always be true, so there wouldn't be a need for the conditional.

Reply:
We will will use the old tests in nightly, so `replace_bootloader` will be false there